pkg/handler/bankaccount/bank_account.go

```diff
@@ -34,11 +34,12 @@ func New(store *store.Store, repo store.DBRepo, service *service.Service, logger
 // List godoc
 // @Summary Get all bank accounts
 // @Description Get all bank accounts
+// @id listBankAccounts
 // @Tags Bank
 // @Accept  json
 // @Produce  json
-// @Success 200 {object} []view.ListBankAccountResponse
-// @Failure 500 {object} view.ErrorResponse
+// @Success 200 {object} ListBankAccountResponse
```

Comment:
@namnhce is this right?

Reply:
I mean why we have the different between 2 of them: array and object

Reply:
value []view.ListBankAccountResponse was wrong, the response of the function is {Data []BankAccount `json:"data"`} = ListBankAccountResponse so I updated it to new model ListBankAccountResponse